src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -37,7 +37,8 @@ const UsersFragment = ({
   withoutRecent = false,
   showRecentSearch = true,
   fetchData = () => {},
-  searchText
+  searchText,
+  isUser
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A new prop `isUser` is introduced here. Ensure that this prop is passed correctly from the parent component and its purpose is well-documented.
```diff
  fetchData = () => {},
  searchText
+ isUser
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->